syntax = "proto3";

package besttick;

message TickerInfo {
  string instID = 1;
  double bestBid = 2;
  double bidSz = 3;
  double bestAsk = 4;
  double askSz = 5;
  int64 eventTs = 6;
  int64 updateID = 7;
}